agot-bg-website/agotboardgame_main/views.py

```diff
@@ -223,6 +223,7 @@ def games(request):
         game.owner = request.user
         game.save()
 
+        # Why this length check? I always wondered why I just sometimes were redirected to the game automatically...
```

Comment:
To avoid too short or too long names, I guess. I don't know I put it there without any error message, though... 🙁